fili-core/src/test/groovy/com/yahoo/bard/webservice/async/AsyncDruidSendsErrorSpec.groovy

```diff
@@ -75,6 +77,33 @@ class AsyncDruidSendsErrorSpec extends AsyncFunctionalSpec {
                         "druidQuery" : null
                     }"""
 
+    final CountDownLatch jobMetadataReady = new CountDownLatch(1)
+
+    def setup() {
+        TestAsynchronousWorkflowsBuilder.addSubscriber(
+                TestAsynchronousWorkflowsBuilder.Workflow.JOB_MARKED_COMPLETE,
+                new Observer() {
```

Comment:
Note that this suggestion follows into the other spec that gets updated in this PR also.